### ISSUE TITLE
Setting Cropped image to input type file?

### ISSUE THREAD

**Issue by Diwas777** [zhanziyang/vue-croppa#163]
Suppose I have hidden input file field like this:

> `<input type="hidden" id="croppedFile">`

Then can i save cropped image to this hidden field on submitting the form like this

> methods:{
> onSubmit(){
> document.getElementById('croppedFile').value=cropped image goes here..
> ...'
> }
> }

Mission: I want to hold cropped image in file . I tried formdata concepts and it works but i need it as above. Is it possible?

**Comment by zhanziyang** [comment]
I don't think it is possible. You can't set the value of file input programmatically except setting `null` to clear it. The browsers do not allow it due to security reasons.

**Comment by Diwas777** [comment]
can't we do something like this:

> vm.cropImg = vm.$refs.croppa.getCroppedCanvas().toDataURL();
> document.getElementById("croppedFile").value = vm.cropImg;

**Comment by Diwas777** [comment]
Ok I got it blob data can only be passed from client side to server side via formdata. And I did the same .
Thank you @zhanziyang

**Comment by zhanziyang** [comment]
Web browsers do not allow programmatically assigning values to an `<input type="file">` for security reasons.

This is outside what the component can control, so we're closing the issue. Instead of populating an input element, upload the exported Blob directly using `FormData`.
